code-samples/deployment/docker/create-a-dockerfile/index.js

```diff
@@ -0,0 +1,26 @@
+const { Client } = require('discord.js');
+require('dotenv').config();
+
+const commands = [
+	{
+		name: 'ping',
+		description: 'Pong!',
+	},
+];
+
+const client = new Client().once('ready', () => {
+	console.log('Ready!');
+
+	client.application.commands.fetch().then(existingCommands => {
+		existingCommands.find(c => c.name === 'ping')
+			&& client.application.commands.set(commands);
+	});
+});
```

Comment:
this type of code is not present anywhere else in the guide, either deploy-commands should be used or the code sample should be removed completely

Reply:
i prefer an easy but complete baseline pingpong bot setup here, to be able and easily maintain samples without having to remember to update a bot for the deployment section

but we should definitely not recommend deploying commands on ready.

Reply:
Yeah I agree, that's why I made it check if it already exists like frameworks such as Sapphire.

If you don't think we should do this, then the only way I could think of to deploy commands at runtime would be pinging with a deploy command.

Reply:
Is it really necessary to have this code sample though, using docker to run a bot does not need any code changes and pm2 section also does not has any simple samples as this, moreover this code isn't referenced anywhere in the actual markdown pages